content/documentation/pages/6-recipes/7-scaling/0-index.md

```diff
@@ -0,0 +1,29 @@
+---
+path: 'recipes/scaling/'
+title: 'Scaling'
+description: 'Using Prometheus and Data Flow Scale API for autoscaling pipelines'
+summary: true
+---
+
+# Scale-Out Data Pipelines
+
+[Scalability](https://en.wikipedia.org/wiki/Scalability) is the ability of a system to cope with increased load by adding resources. The [vertical](https://en.wikipedia.org/wiki/Scalability#Vertical) scaling (up/down) adds resources such as CPUs, memory and storage to the `existing nodes`, while the [horizontal](https://en.wikipedia.org/wiki/Scalability#Horizontal) scaling (out/in) adds `additional nodes` to the system.
+Following recipes demonstrate how to use `horizontal scaling` (e.g. `scaling out` or `shared nothing`) for streaming data pipelines.
+
+In Spring Cloud Data Flow the [stream data processing](%currentPath%/concepts/streams/) is implemented as a collection of independent event-driven streaming applications (microservices) that connect using a messaging middleware such as RabbitMQ, Apache Kafka.
+The ability of a streaming pipeline to cope with increased data load depends on the:
+
+- _Messaging Middleware Scalability_ - [data partitioning](https://docs.spring.io/spring-cloud-stream/docs/current/reference/htmlsingle/#partitioning) is a widely used technique for scaling messaging middleware and Data Flow provides extensive [stream partition support](%currentPath%/feature-guides/streams/partitioning/#data-partitioning/).
+- _Event-driven Applications Scalability_ - Spring Cloud Data Flow provides event-driven applications parallelism, referred to as application scaling. Later allows to allocate multiple application instances, usually one for every data partition topic.
+
+Following diagram illustrates a typical scale-out architecture based on data partitioning and applications parallelism:
+
+![SCDF Stream Scaling](images/scdf-stream-partition-and-scaling-overview.png)
+
+Platforms such as Kubernetes and Cloud Foundry offer scaling features for the TechOps teams to control system’s load. For example TechOps can leverage [CF Scale](https://docs.cloudfoundry.org/devguide/deploy-apps/cf-scale.html), [scale applications](https://docs.spring.io/spring-cloud-dataflow/docs/2.3.0.RELEASE/reference/htmlsingle/#configuration-cloudfoundry-scaling) or [Kubectl Scale](https://jamesdefabia.github.io/docs/user-guide/kubectl/kubectl_scale/) to scale applications manually or enable autoscaling features such as [App Autoscaler](https://docs.run.pivotal.io/appsman-services/autoscaler/using-autoscaler.html) for Cloud Foundry and [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) and [KEDA](https://github.com/kedacore/keda) for Kubernetes. The autoscaling is usually controlled by CPU or memory limit thresholds (HPA) or message queues sizes (KEDA).
```

Comment:
I never heard of the term `TechOps`. I don't think it is commonly used in the enterprise. It might be confusing if we go after something that is uncommon in the industry.

Reply:
Perhaps just call them as "Operators" — this is well understood and is very common.